Source/FormTextField.swift

```diff
@@ -86,28 +85,16 @@ public class FormTextField: UITextField, UITextFieldDelegate {
     private lazy var customClearButton: UIButton? = {
         var button: UIButton? = nil
 
-        if self.isUsingClearButton {
-            let image = FormTextFieldClearButton.imageForSize(CGSize(width: 18, height: 18), color: self.accessoryButtonColor)
-            button = UIButton(type: .Custom)
-            button?.setImage(image, forState: .Normal)
-            button?.addTarget(self, action: #selector(FormTextField.clearButtonAction), forControlEvents: .TouchUpInside)
-            button?.frame = CGRect(x: 0, y: 0, width: FormTextField.AccessoryButtonWidth, height: FormTextField.AccessoryButtonHeight)
-        }
+        let image = FormTextFieldClearButton.imageForSize(CGSize(width: 18, height: 18), color: self.accessoryButtonColor)
+        button = UIButton(type: .Custom)
+        button?.setImage(image, forState: .Normal)
+        button?.addTarget(self, action: #selector(FormTextField.clearButtonAction), forControlEvents: .TouchUpInside)
+        button?.frame = CGRect(x: 0, y: 0, width: FormTextField.AccessoryButtonWidth, height: FormTextField.AccessoryButtonHeight)
 
         return button
     }()
 
-    private lazy var inputValidationIndicator: UIImageView? = {
-        var imageView: UIImageView? = nil
-
-        if let image = self.accessoryImage {
-            imageView = UIImageView(image: image)
-            imageView?.contentMode = .Center
-            imageView?.frame = CGRect(x: 0, y: 0, width: FormTextField.AccessoryButtonWidth, height: FormTextField.AccessoryButtonHeight)
-    }
-
-        return imageView
-    }()
+    public var accessoryView: UIView?
```

Comment:
I like it because then we have an optional accessoryView, and an optional clearButton!